sphinx_js/jsdoc.py

```diff
@@ -156,6 +156,10 @@ def is_private(doclet):
     return doclet.get('access') == 'private'
 
 
+def is_static(obj):
+    return obj.get('scope', '') == 'static'
```

Comment:
I understand the desire to have get() return a consistent type here, but we should either make it like is_private() (which says just `doclet.get('access')`) or change is_private() to be like `get('access', '')` for consistency.

Reply:
Think so too. For consistency, I changed it to be like the is_private method. Basically, there's no change for us if we compare None or '' with 'static' 👍. I also changed the parm ``obj`` to ``doclet``, so we're even more consistent.